collect_app/src/main/java/org/odk/collect/android/preferences/qr/QRScannerFragment.java

```diff
@@ -24,43 +26,70 @@
 
 import com.google.zxing.ResultPoint;
 import com.google.zxing.client.android.BeepManager;
+import com.google.zxing.client.android.Intents;
+import com.google.zxing.integration.android.IntentIntegrator;
 import com.journeyapps.barcodescanner.BarcodeCallback;
 import com.journeyapps.barcodescanner.BarcodeResult;
+import com.journeyapps.barcodescanner.CaptureManager;
 import com.journeyapps.barcodescanner.DecoratedBarcodeView;
+import com.journeyapps.barcodescanner.camera.CameraSettings;
 
 import org.odk.collect.android.R;
 import org.odk.collect.android.preferences.utilities.SettingsUtils;
+import org.odk.collect.android.utilities.CameraUtils;
 import org.odk.collect.android.utilities.CompressionUtils;
 import org.odk.collect.android.utilities.ToastUtils;
 
 import org.odk.collect.android.listeners.PermissionListener;
 import org.odk.collect.android.utilities.PermissionUtils;
+import org.odk.collect.android.utilities.WidgetAppearanceUtils;
 
 import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
 import java.util.List;
 import java.util.zip.DataFormatException;
 
 public class QRScannerFragment extends Fragment implements DecoratedBarcodeView.TorchListener {
```

Comment:
This should be in a different package, now. It's also no longer just a QR code scanner.

Reply:
Right, fixed.